src/test/java/uk/gov/hmcts/reform/sandl/snlrules/models/HearingPartTests.java

```diff
@@ -37,7 +37,7 @@ public void toDescription_returnsProperlyFormattedString_whenAllFieldsAreGiven()
         testObject.setScheduleStart(OffsetDateTime.parse("2018-09-26T14:00:00.000Z"));
         testObject.setScheduleEnd(OffsetDateTime.parse("2018-09-26T16:00:00.000Z"));
 
-        String expected = "Duration: 01:02, Case type: ct, Hearing type: ht,"
+        String expected = "Duration: 01:02, Case type code: ct, Hearing type code: ht,"
```

Comment:
Probably question to @kpakur . should I return case type code in the description?

Reply:
yes code, due to 2 reasons: 
1. rules do have only code, not description (should not have as this is more memory consumption and more computing at then were we are optimizing for this)
2. the description from rules in frontend is temporary for build extension and will be done in UI post-build extension
3. It's been agreed with Teresa and Sophie that we use just the code to not spend time on extending feature (description from rules) that will be replaced long term (with the forntend generated description) and they are ok with some limitation of this approach